libkbfs/folder_branch_ops.go

```diff
@@ -4704,6 +4704,11 @@ func (fbo *folderBranchOps) applyMDUpdatesLocked(ctx context.Context,
 				return err
 			}
 		}
+		if rmd.IsRekeySet() {
```

Comment:
But this PR doesn't trigger a rekey immediately on MD updates anymore; instead, it only unsets the paper prompts. Could you be more specific how the device's own update can interfere it?